app/src/androidTest/java/fiofoundation/io/fiokotlinsdktestapp/TestNetSdkTests.kt

```diff
@@ -406,14 +452,17 @@ class TestNetSdkTests {
             throw AssertionError("Pending Requests Failed: " + generalException.message)
         }
 
+        // this approach to testing, or getting the request id 1 will not work on test net, this test needs work
+        // it works well when run on a local 3 node test net, but will not work when run on testnet.
         println("testFundsRequest: Test recordObtData")
         try
         {
             val pendingRequests = this.bobFioSdk!!.getPendingFioRequests()
 
             if(pendingRequests.isNotEmpty())
             {
-                val firstPendingRequest = pendingRequests.firstOrNull{it.payerFioAddress == this.bobFioAddress}
+                val firstPendingRequest = pendingRequests.firstOrNull{( it.fioRequestId == BigInteger("1")
```

Comment:
this PR is set to merge with Master.  Are you sure that is what is wanted since there is “temporary” code still there?

see comment on line 455.